### ISSUE TITLE
DD4hep: PixelForwardBlade copy numbers are off

### ISSUE THREAD

**Issue by ianna** [cms-sw/cmssw#28261]
DD4hep geometry: the green volume should have a copy number 55. See reference snapshot.
![Screen Shot 2019-10-24 at 17 53 53](https://user-images.githubusercontent.com/1390682/67503390-d8cf6880-f687-11e9-9ba2-fcf32ac1e1aa.png)
DD geometry reference:
![Screen Shot 2019-10-24 at 17 54 10](https://user-images.githubusercontent.com/1390682/67503415-e4229400-f687-11e9-9bb2-2008ccc0db0d.png)

Hint: inner volume copies start at 0 and end at 21, outer volume copies should start at 22( not 0!)

The following error should disappear when PixelForwardBlade volumes are numbered correctly.
```
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_0
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_1
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_2
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_3
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_4
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_5
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_6
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_7
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_8
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_9
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_10
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_11
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_12
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_13
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_14
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_15
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_16
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_17
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_18
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_19
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_20
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_21
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_0
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_1
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_2
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_3
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_4
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_5
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_6
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_7
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_8
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_9
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_10
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_11
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_12
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_13
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_14
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_15
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_16
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_17
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_18
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_19
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_20
PlacedVolume     ERROR ++ Attempt to add already exiting node PixelForwardBlade_21
```

**Comment by ianna** [comment]
@vargasa - Could you, please, have a look? Thanks.

**Comment by vargasa** [comment]
Thanks @ianna. I'll take a look. I have seen this in one of the migrated algorithms

**Comment by ianna** [comment]
fixed in #28378
